src/terms/operators.jl

```diff
@@ -108,6 +110,50 @@ function apply!(Hψ, op::NonlocalOperator, ψ)
 end
 Matrix(op::NonlocalOperator) = op.P * op.D * op.P'
 
+struct NonlocaldOperator{T <: Real, PT, DT} <: RealFourierOperator
```

Comment:
comment

Reply:
I don't think they should be a RealFourierOperator

Reply:
Hm, do you really need to have a specific method for this? Can't you see it as a combination a multiply_by_exp_iqr and a periodic operator? We should discuss